src/vs/editor/common/standaloneStrings.ts

```diff
@@ -26,6 +26,8 @@ export namespace AccessibilityHelpNLS {
 	export const listAlerts = nls.localize("listAnnouncementsCommand", "Run the command: List Signal Announcements for an overview of announcements and their current status.");
 	export const quickChat = nls.localize("quickChatCommand", "Toggle quick chat<keybinding:workbench.action.quickchat.toggle> to open or close a chat session.",);
 	export const startInlineChat = nls.localize("startInlineChatCommand", "Start inline chat<keybinding:inlineChat.start> to create an in editor chat session.");
+	export const setBreakpoint = nls.localize('debugConsole.setBreakpoint', "The Debug: Inline Breakpoint command<keybinding:editor.debug.action.toggleInlineBreakpoint> will set or unset a breakpoint at the current cursor position.");
```

Comment:
You probably don't want the command ID or the special syntax to go through localization

Reply:
Hm, so would I pass those in as variables to the localization string?

Reply:
You can use the {0} template syntax and pass in an unlocalized string

Reply:
#221843